### ISSUE TITLE
Use absolute import in dipy/core

### ISSUE THREAD

**Issue by sahmed95** [nipy/dipy#994]
Please refer to the original issue #969. Use absolute import in place of relative for all .py files inside /dipy. Take care not to change .pyx or **init** files 

``` python
/dipy/dipy/core/gradients.py:
    1  from __future__ import division, print_function, absolute_import
    2  
    3: from ..utils.six import string_types
    4  
    5  import numpy as np
    .
   10  from scipy.linalg import inv
   11  
   12: from ..io import gradients as io
   13: from .onetime import auto_attr
   14: from .geometry import vector_norm
   15  
   16  

/dipy/dipy/core/profile.py:
    5  import subprocess
    6  
    7: from ..utils.optpkg import optional_package
    8  
    9  cProfile, _, _ = optional_package('cProfile')

/dipy/dipy/core/sphere.py:
    4  import warnings
    5  
    6: from ..utils.six.moves import xrange
    7  
    8  from dipy.core.geometry import cart2sphere, sphere2cart, vector_norm

/dipy/dipy/core/subdivide_octahedron.py:
   11  
   12  """
   13: from .sphere import unit_octahedron, HemiSphere
   14  
   15  def create_unit_sphere( recursion_level=2 ):

/dipy/dipy/core/tests/test_sphere.py:
    5  import warnings
    6  
    7: from ...utils.six.moves import xrange
    8  
    9  from dipy.core.sphere import (Sphere, HemiSphere, unique_edges, unique_sets,
```

**Comment by souravsingh** [comment]
#992

**Comment by skoudoro** [comment]
Fix by #992 . Closing
